components/org.wso2.carbon.identity.workflow.engine/src/main/java/org.wso2.carbon.identity.workflow.engine/DefaultApprovalWorkflow.java

```diff
@@ -0,0 +1,33 @@
+package org.wso2.carbon.identity.workflow.engine;
+
+import org.wso2.carbon.identity.workflow.mgt.bean.Parameter;
+import org.wso2.carbon.identity.workflow.mgt.bean.metadata.InputData;
+import org.wso2.carbon.identity.workflow.mgt.bean.metadata.ParameterMetaData;
+import org.wso2.carbon.identity.workflow.mgt.exception.WorkflowException;
+import org.wso2.carbon.identity.workflow.mgt.exception.WorkflowRuntimeException;
+import org.wso2.carbon.identity.workflow.mgt.workflow.AbstractWorkflow;
+import org.wso2.carbon.identity.workflow.mgt.workflow.WorkFlowExecutor;
+
+import java.util.List;
+
+public class DefaultApprovalWorkflow extends AbstractWorkflow {
```

Comment:
Add class comments. Let's fix other places as well

Reply:
Addresses with this commit https://github.com/wso2-incubator/identity-workflow-simple/pull/1/commits/0f8bc6df9d488542344585725d992c952c1b640b